api/envoy/config/route/v3/route_components.proto

```diff
@@ -760,6 +760,28 @@ message RouteAction {
     ConnectConfig connect_config = 3;
   }
 
+  message MaxStreamDuration {
+    // Specifies the maximum duration allowed for streams on the route. If not specified, the
+    // value from :ref:`max_stream_duration
+    // <envoy_api_field_config.core.v3.HttpProtocolOptions.max_stream_duration>` is used.
```

Comment:
I think this should say something like this:

```
If not specified, the value from the :ref:`max_stream_duration
<envoy_api_field_config.core.v3.HttpProtocolOptions.max_stream_duration>` field
in :ref:`HttpConnectionManager.common_http_protocol_options
<envoy_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.common_http_protocol_options>` is used.
```

Reply:
Done.